src/pages/DetailsPage/DetailsPage.spec.js

```diff
@@ -50,11 +52,41 @@ describe('DetailsPage', () => {
           },
         }}
         onClickDetailsBack={onClickDetailsBack}
+        onToggleFavorite={jest.fn()}
+        isFavorite={false}
       />
     )
 
-    const back = screen.getByRole('button')
+    const back = screen.getByTestId('back')
     userEvent.click(back)
     expect(onClickDetailsBack).toHaveBeenCalled()
   })
+  it('should call onToggleFavorites with argument id', () => {
```

Comment:
Not sure if this test is needed as you already test it in the button spec, but really not 100% sure.